Sources/MapboxDirections/MapboxStreetClass.swift

```diff
@@ -1,65 +1,68 @@
 
 import Foundation
 
+/// 'Mapbox Streets V8' compliant street class definition
+///
+/// For detailed info on road classes, see [the documentation](https://docs.mapbox.com/vector-tiles/reference/mapbox-streets-v8/#road)
 public enum MapboxStreetClass: String, Codable {
```

Comment:
Sorry I didn’t catch this earlier, but it should be MapboxStreetsRoadClass, as wordy as that is. Even the existing RoadClass is Mapbox-specific; what’s new about this enumeration is that it’s based on the Mapbox Streets source. Alternatively, if that sounds too wordy, we can call it DetailedRoadClass. Just make sure to be consistent throughout the PR.